pkg/apis/kops/model/features.go

```diff
@@ -31,6 +31,8 @@ func UseChallengeCallback(cloudProvider kops.CloudProviderID) bool {
 		return true
 	case kops.CloudProviderScaleway:
 		return true
+	case kops.CloudProviderAzure:
+		return true
```

Comment:
Does this require additional NetworkSecurityGroup rules to allow access from the control plane VMs to the nodes?

Reply:
Was just looking at that. Updated with a new rule to allow it.